src/services/peerconnection.ts

```diff
@@ -135,6 +138,12 @@ export class PeerConnectionHelper {
         };
         this.webrtcTask.on('candidates', (e: saltyrtc.tasks.webrtc.CandidatesEvent) => {
             for (let candidateInit of e.data) {
+                if (candidateInit) {
+                    this.$log.debug(this.logTag, 'Adding remote ICE candidate:',
+                        PeerConnectionHelper.censorCandidate(candidateInit.candidate));
+                } else {
+                    this.$log.debug(this.logTag, 'No more remote ICE candidates');
```

Comment:
Huh, you're right, we haven't specified that in SaltyRTC's WebRTC task. However, WebRTC does use `null` to indicate end of candidates (as can be seen in line 98) and we actually should pass that information to the stack to possibly improve the gathering process.

By the way, it has been specified for the [SaltyRTC ORTC Task](https://github.com/saltyrtc/saltyrtc-meta/blob/master/Task-ORTC.md#ice-candidates-message).